docs/getting-started.md

```diff
@@ -41,24 +41,37 @@ Antrea deployment manifest:
 
 ## Theia Installation
 
+In Theia v0.1.0, we introduce the Flow Aggregator Helm Chart to Theia
+From Theia v0.2.0 and Antrea v1.8.0, the Flow Aggregator Helm Chart is moved from Theia to Antrea.
```

Comment:
Line wrapping.

Reply:
I forgot to remove these sentences after I rephrase them here:
https://github.com/antrea-io/theia/blob/3fa731ca4527c97fbfa0286d47590c89252e1faf/docs/getting-started.md?plain=1#L50-L58
Removed them. Please tell me if you think it is unclear.